src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.AnyMobile.cs

```diff
@@ -796,6 +797,8 @@ private void ThrowForModifiedManagedSslOptionsIfStarted()
             _socketHandler!.SslOptions = _socketHandler!.SslOptions;
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2075:GetMethod",
+            Justification = "The DynamicDependency declarations will ensure the methods will be preserved.")]
```

Comment:
Where are these DynamicDependency declaration? I don't see them in this file. It might be good to point to them in the Justification.

Or even better, annotate this method as `RequiresUnreferencedCode` and then suppress the warnings at the callsites that have the DynamicDependency attributes on them.

Reply:
Thanks for the comment. All the call sites were in a separate file - `HttpClientHandler.AnyMobile.InvokeNativeHandler.cs`. I moved the method into that file so that they are right next to each other.

I'm not sure if the `RequiresUnreferencedCode` attribute would still make sense after the last change based on Michal's suggestions.

Reply:
The comment was applicable to the previous iteration only. The code is trim safe by construction now.